src/client/src/rt-system/AutoBahnConnection.ts

```diff
@@ -1,8 +1,12 @@
 import { Connection } from 'autobahn'
```

Comment:
Instead of editing AutobahnConnection interface and implementation, can we create a "new" interface without the name Autobahn in it? Everything else stays the same. E.g. just `Connection` for the interface, and `RabbitStompConnection` for the implementation, or something like that...